test/unit_test/agent/sandbox/test_ucloud_agent_sandbox_provider.py

```
@@ -0,0 +1,358 @@
+
+
+class _FakeFiles:
+    def __init__(self):
+        self.files: dict[str, bytes] = {}
+        self.list_overrides: dict[str, list[SimpleNamespace]] = {}
+
+    def write(self, path: str, data: str | bytes, request_timeout=None):
+        self.files[path] = data.encode("utf-8") if isinstance(data, str) else data
+        return SimpleNamespace(path=path)
+
+    def read(self, path: str, format: str = "text", request_timeout=None):
```

Ruff (0.16.1)

[error] Function argument `format` is shadowing a Python builtin

(A002)